index.js

```diff
@@ -20,7 +20,7 @@ const removeUnusedMenuItems = menuTemplate => {
 	let notDeletedPreviousElement;
 
 	return menuTemplate
-		.filter(menuItem => menuItem !== undefined && menuItem !== false && menuItem.visible !== false)
+		.filter(menuItem => menuItem !== undefined && menuItem !== false && menuItem.visible !== false && menuItem.visible !== '')
```

Comment:
I just realized that this is fixed in the wrong place. `.visible` is expected to be a boolean. This should be fixed everywhere we use `.visible` to ensure it always results in being a boolean. For example, here: https://github.com/sindresorhus/electron-context-menu/blob/307d6d4dc1dcfeb68a8c7365499828174592451a/index.js#L215

However, we can keep this workaround in case the user accidentally does the same, but we should at least do the right thing.